modules/vscode/hm.nix

```
@@ -62,5 +70,26 @@ in
             "screencastMode.fontSize" = sizes.terminal * 4.0 / 3.0 * 56.0 / 14.0;
           };
         };
-      };
+      }) cfg.profileNames
+    );
+    warnings =
+      lib.mkIf
+        (
+          config.programs.vscode.enable
+          && config.stylix.targets.vscode.profileNames == [ ]
+        )
+        [
+          ''
+            VSCode is currently enabled in both home-manager and Stylix, but Stylix
+            has no profile names to theme. These can be set with
+
+                config.stylix.targets.vscode.profileNames = [ "nameOfProfile" ];
+
+            Alternatively, this warning can be disabled by disabling the VSCode
+            Stylix module
+
+                config.stylix.targets.vscode.enable = false; 
+          ''
+        ];
```

What about the following adapted warning:

```suggestion
      lib.optional
        (
          config.programs.vscode.enable
          && config.stylix.targets.vscode.profileNames == [ ]
        )
        ''stylix: vscode: `config.stylix.targets.vscode.profileNames` is not set. Declare profile names with 'config.stylix.targets.vscode.profileNames = [ "<PROFILE_NAME>" ];'.'';
```

None of the existing Stylix warnings mention how to disable them, since disabling the module seems somewhat obvious.